queries/c_sharp/highlights.scm

```diff
@@ -0,0 +1,137 @@
+
+(method_declaration
+  name: (identifier) @method)
+
+(parameter_list
+  (parameter
+   name: (identifier) @parameter))
+
+[
+(integer_literal)
+(real_literal)
+] @number
+
+(null_literal) @constant.builtin
+(character_literal) @character
+
+[
+(string_literal)
+(interpolated_string_text)
+] @string
+(boolean_literal) @boolean
+
+[
+(predefined_type)
+(implicit_type)
+(void_keyword)
+ ] @type.builtin
+
+(comment) @comment
+
+(using_directive
+  (identifier) @type)
+
+(using_directive
+  (qualified_name
+    (identifier) @type))
+
+(interface_declaration
+  name: (identifier) @type)
+(class_declaration
+  name: (identifier) @type)
+(enum_declaration
+  name: (identifier) @type)
+(constructor_declaration
+  name: (identifier) @type)
+(variable_declaration
+  (identifier) @type)
+
+(object_creation_expression
+  (identifier) @type)
+
+(attribute
+ name: (identifier) @attribute)
+
+[
+"if"
+"else"
+"switch"
+"case"
+] @conditional
+
+[
+  "while"
+  "for"
+  "do"
+  "continue"
+  "break"
+  "goto"
+  "foreach"
+] @repeat
+
+[
+ "try"
+ "catch"
+ "throw"
+ "finally"
+] @exception
+
+[
+ "abstract"
+ "const"
+ "extern"
+ "internal"
+ "override"
+ "private"
+ "protected"
+ "public"
+ "readonly"
+ "sealed"
+ "static"
+ "virtual"
+ "volatile"
+ "async"
+ "await"
+] @StorageClass
+
```

Comment:
StorageClass?

Reply:
Oh is that not a valid syntax type?  Ok I've changed this to just `keyword` which seems to be more in line with what the other files are doing